src/hiveboard_bridge/src/main.cpp

```diff
@@ -0,0 +1,34 @@
+#include "../include/hiveboard_bridge/TCPServer.h"
+#include "../include/hiveboard_bridge/TCPServerMonitor.h"
+#include "ros/ros.h"
+
+int main(int argc, char** argv) {
+    ros::init(argc, argv, "hiveboard_bridge");
+
+    TCPServerMonitor monitor;
+    TCPServer socket(8080, monitor);
```

Comment:
Will it always be port 8080?

Reply:
This should be provided by the launch file. In the case we launch multiple nodes, each node should have their own port

Reply:
added a launch file.